app/models/phase.rb

```diff
@@ -98,8 +98,7 @@ def deep_copy(**options)
   # TODO: Move this to Plan model as `num_answered_questions(phase=nil)`
   # Returns the number of answered question for the phase.
   def num_answered_questions(plan)
-    return 0 if plan.nil?
-    sections.to_a.sum { |s| s.num_answered_questions(plan) }
+    plan&.num_answered_questions.to_i
```

Comment:
You just changed my life :)

I grew very attached to this safe nav approach when working with Groovy years ago `obj?.method`. Glad to see its now a part of Ruby!

Reply:
Yep. It's not the prettiest syntax, but a very useful tool!